src/server/index.ts

```diff
@@ -255,6 +263,10 @@ export class Server<
     return this._capabilities;
   }
 
+  private getInstructions(): string | undefined {
+    return this._instructions;
+  }
```

Comment:
Can we please remove this? I don't think there's any benefit to a private getter in this case.